starter-kits/nextjs-starter-kit/primitives/Popover/Popover.module.css

```diff
@@ -29,7 +29,6 @@
   border-radius: var(--radius);
   box-shadow: var(--shadow-xl);
   animation: 0.15s ease-in-out;
-  overflow: hidden;
```

Comment:
Copy-paste conflict, this was added in https://github.com/liveblocks/liveblocks/pull/1437 and shouldn't be removed.